validation-test/SDK/Inputs

```diff
@@ -0,0 +1 @@
+../../test/sdk/Inputs
```

Comment:
Should this symbolic link be `../../test/SDK/Inputs` for case-sensitive volumes?